content/en/tracing/troubleshooting/_index.md

```diff
@@ -2,444 +2,109 @@
 
-```shell
-[Mon MM  DD 19:56:23 YYYY] [YYYY-MM-DDT19:56:23+00:00] [ddtrace] [debug] - About to send trace(s) to the agent
 ```
-
-
-### Trace successfully sent to the Agent
-
-```shell
-[Mon MM  DD 19:56:23 2019] [YYYY-MM-DDT19:56:23+00:00] [ddtrace] [debug] - Traces successfully sent to the agent
+ERROR | (pkg/trace/logutil/throttled.go:38 in log) | http.Server: http: Accept error: request has been rate-limited; retrying in 80ms
 ```
 
 
-{{% /tab %}}
-{{< /tabs >}}
+To increase the APM connection limit for the Agent, configure the `connection_limit` attribute within the Agent's configuration file (underneath the `apm_config:` section). For containerized deployments (Docker, Kubernetes, etc.), use the `DD_APM_CONNECTION_LIMIT` environment variable.
 
+## Troubleshooting data requested by Datadog Support
 
-## APM rate limits
+When you open a [support ticket][1], our support team may ask for a combination of the below information:
 
-### Max events per second limit
+1. How you are confirming the issue (links to a trace, screenshots, etc) and what you expect to see
 
-If you encounter the following error message in your Agent logs, your application(s) are emitting more than the default 200 trace events per second allowed by APM.
+    This allows us to confirm errors and attempt to reproduce your issues within our testing environments.
 
-```
-Max events per second reached (current=300.00/s, max=200.00/s). Some events are now being dropped (sample rate=0.54). Consider adjusting event sampling rates.
+1. [Tracer Startup Logs](#tracer-startup-logs)
 
-```
+    Startup logs are a great way to spot misconfiguration of the tracer, or the inability for the tracer to communicate with the Datadog Agent.  By comparing the configuration that the tracer sees to the one set within the application or container, we can identify any areas where a setting is not being properly applied.
 
+1. [Tracer Debug Logs](#tracer-debug-logs)
 
-To increase the APM rate limit for the Agent, configure the `max_events_per_second` attribute within the Agent's configuration file (underneath the `apm_config:` section). For containerized deployments (Docker, Kubernetes, etc.), use the `DD_APM_MAX_EPS` environment variable.
+    Tracer Debug logs go one step deeper than startup logs, and will help us to identify if integrations are instrumenting properly in a manner that we aren't able to necessarily check until traffic flows through the application.
 
-**Note**: Increasing the APM rate limit could result in increased costs for App Analytics.
+1. An [agent flare][4] from your agent in [debug mode][5]
 
+    Agent flares allow us to see what is happening within the Datadog Agent, if any traces are being rejected or malformed within the Agent.  This will not help if traces are not reaching the Agent, but does help us identify the source of an issue, as well as identifying any metric discrepancies.
```

Comment:
I wonder if this might be a time to mention the `TRACE` level logging as that does help us resolve issues with deeper logging than `DEBUG`.  I suspect we don't advertise `TRACE` more often because it's very verbose logging but this might be the right circumstance to mention it.  Perhaps it's discussing with other PMs and/or the agent team.